xbmc/interfaces/json-rpc/PVROperations.cpp

```diff
@@ -117,10 +118,10 @@ JSONRPC_STATUS CPVROperations::GetChannels(const std::string &method, ITransport
     return InvalidParams;
 
   CFileItemList channels;
-  const std::vector<std::shared_ptr<PVRChannelGroupMember>> groupMembers = channelGroup->GetMembers(CPVRChannelGroup::Include::ONLY_VISIBLE);
+  const auto groupMembers = channelGroup->GetMembers(CPVRChannelGroup::Include::ONLY_VISIBLE);
```

Comment:
Can it be safe to use only his related pointer? If yes maybe to add the `&` to prevent data copies.

Reply:
No. Look at the implementation of `CPVRChannelGroup::GetMembers`. It does not simply return a vector containing the channel group members. The returned vector is compiled at runtime according to the filter passed in.

Reply:
Aha and OK, makes then no sense and can't work :smile:. Thanks for the note.